server/src/memes.ts

```diff
@@ -29,7 +29,7 @@ memes.post('/', upload, async (req: Request, res: Response) => {
       tags: req.body.tags,
       url: makeUploadUrl(req.file.filename),
     });
-    res.sendStatus(201).send(meme);
+    res.status(201).send(meme);
   } catch (e) {
     logger.error(e);
     res.sendStatus(500);
```

Comment:
@mikitachab do we need to use ```sendStatus``` instead ```status``` in catch block?   
Is it correct that we send status (not like in a try block)?

Reply:
actually reason of error was that response was sent after `sendStatus` without body, and then `.send(meme) ` tried to send response again. In a catch block we do not want to send any data, just immediately send response with status 500.  So i believe that everything is correct.

Reply:
👍